### ISSUE TITLE
Customizing the embed (esp. scale)

### ISSUE THREAD

**Issue by pixlpa** [sammdec/nfte#35]
I'd like to create a customized embed (using the React component). My primary interest is creating a custom box size, but exposing other style/layout CSS features to the top level component would also be really helpful.
If I understand correctly, the only way to change embed size currently is to copy the contents of components/Embed.jsx into my own custom component, since the max-height/max-width and aspect values aren't exposed to the top level props. Maybe there's a way to achieve this I haven't yet discovered?

**Comment by sammdec** [comment]
Yeah that’s correct, the easiest way is to duplicate and add your own customisations to the render component.
